src/main/resources/mcmod.info

```diff
@@ -11,6 +11,6 @@
       "credits": "",
       "logoFile": "assets/refinedstorage/textures/logo.png",
       "screenshots": [],
-      "dependencies": []
+      "dependencies": ["inventorytweaks"]
```

Comment:
How is it not?  "dependencies | [string] | A list of modids." And "inventorytweaks" is the modid for Inventory Tweaks as can be seen here https://github.com/Inventory-Tweaks/inventory-tweaks/blob/develop/src/main/resources/mcmod.info , so what am I missing?